### DISCUSSION TITLE
how to install ?? to use on chrome please help

### DISCUSSION THREAD

**Discussion by jankiki87** [QIN2DIM/hcaptcha-challenger#160] [Q&A]
how to install it use in windows in chrome ??

**Comment by QIN2DIM** [comment]
ah??

**Reply by jankiki87** [reply]
![Przechwytywanie](https://user-images.githubusercontent.com/112200369/187652076-b0a26dd0-c3a7-44df-89b0-4b3612ff2bed.PNG)


how to run on specified page ??
